api/AudioScheduledSourceNode.json

```diff
@@ -115,7 +115,7 @@
               "version_added": "18"
             },
             "edge": {
-              "version_added": "≤79"
+              "version_added": "79"
```

Comment:
I'm skeptical about this one since https://developer.mozilla.org/en-US/docs/Web/API/AudioBufferSourceNode#browser_compatibility + https://developer.mozilla.org/en-US/docs/Web/API/OscillatorNode#browser_compatibility were in Edge 12.

Reply:
Surprisingly enough, `AudioScheduledSourceNode` wasn't present in Edge 15 or 18, either with the standard name or the alternate name used in other browsers at first, `AudioSourceNode`.